terrsoshi/tianjiewong-portfolio
# Add Jest Testing Framework to Next.js app

## Description

This PR integrates Jest as the testing framework for the project's Next.js app. This initial setup includes basic configuration and sample tests to ensure everything is working correctly.

## How to Test

1. `cd` into the `app` folder.
2. Run `npm install` to install the new dependencies.
3. Run `npm run test` to execute the sample tests and ensure Jest is correctly set up.

## Additional Notes

- Future test cases can be added in the `__tests__` subdirectory within each respective directory, following the established pattern. (e.g. `src/app/__tests__`, `src/components/__tests__`, etc.)
